extensions/amp-analytics/0.1/events.js

```diff
@@ -171,6 +185,14 @@ function isVideoTriggerType(triggerType) {
   return triggerType.startsWith('video');
 }
 
+/**
+ * @param {string} triggerType
+ * @return {boolean}
+ */
+function isCustomBrowserTriggerType(triggerType) {
+  return Object.values(BrowserEventType).includes(triggerType);
```

Comment:
`includes` is not supported in IE11. Use `indexOf`.

Reply:
Fixed!